removesDup.c

```diff
@@ -0,0 +1,30 @@
+#include <stdio.h>
+
+int main(){
+	int n; // number of elements in array
+	int a[n]; // creates an array
```

Comment:
You got a seg fault! Any idea of how to fix that? Think of what a seg fault is (an error with memory allocation). I'll give you the hint that this is the line where you went wrong - what should you have done differently?